prow/images/gardener-rotate/README.md

```diff
@@ -0,0 +1,13 @@
+# Gardener-rotate image
+
+## Overview
+
+This folder contains the gardener-rotate binary that is used to automatically update Gardener kubeconfig secrets.
```

Comment:
Why we store binary in repo?

Reply:
It was changed to say this folder contains the image (image definition, Dockerfile and makefile)